src/fsharp/FSharp.Core.Unittests/FSharp.Core.Unittests.dll.config

```diff
@@ -5,7 +5,7 @@
         <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
             <dependentAssembly>
               <assemblyIdentity name="FSharp.Core" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
-              <bindingRedirect oldVersion="2.0.0.0-4.4.0.0" newVersion="{FSCoreVersion}"/>
+              <bindingRedirect oldVersion="2.0.0.0-4.4.1.0" newVersion="{FSCoreVersion}"/>
```

Comment:
Dont think so, for example profile 259 set FsCoreVersion to 3.259.41.9055, so ch anging upper bound will restrict binding range